stubs/solvers.py

```diff
@@ -262,25 +261,25 @@ def init_petsc_matnest(self):
             self.Jpetsc_nest = d.PETScNestMatrix(Jpetsc).mat()
             # self.Jpetsc_nest = self.d_to_p(d.PETScNestMatrix(Jpetsc))
         self.Jpetsc_nest.assemble()
-        print(f"Jpetsc_nest assembled, size = {self.Jpetsc_nest.size}")
+        logger.info(f"Jpetsc_nest assembled, size = {self.Jpetsc_nest.size}")
 
     def d_to_p(self, dolfin_matrix):
         return d.as_backend_type(dolfin_matrix).mat()
 
     def init_petsc_vecnest(self):
         dim = self.dim
         if self.print_assembly:
-            fancy_print("Initializing block residual vector", format_type="assembly")
+            logger.info("Initializing block residual vector", extra=dict(format_type="assembly"))
 
         Fpetsc = []
         for j in range(dim):
             Fsum = None
             for k in range(len(self.Fforms[j])):
                 if self.Fforms[j][k].function_space(0) is None:
                     if self.print_assembly:
-                        fancy_print(
+                        logger.info(
```

Comment:
I'm not sure when this would occur, but I think it should be printed as a warning.

Reply:
Fixed in [f8eaa1a](https://github.com/RangamaniLabUCSD/stubs/pull/82/commits/f8eaa1aaec9d83148ef6dac5741a4913832d799b)